### PR TITLE
#864: Added Python 3.10 template flavor

### PR DESCRIPTION
**Pull request by tkilias**
Related to https://github.com/exasol/script-languages-release/issues/864

### CODE REVIEW

File: flavors/template-Exasol-all-python-3.10/flavor_base/build_deps/packages/apt_get_packages [hunk 1]
```diff
@@ -0,0 +1,9 @@
+coreutils|8.28-1ubuntu1
+locales|2.27-3ubuntu1.6
+tar|1.29b-2ubuntu0.4
+curl|7.68.0-1ubuntu2.19
+openjdk-11-jdk|11.0.19+7~us1-0ubuntu1~20.04.1
```
Comment (Nicoretti) [inline flavors/template-Exasol-all-python-3.10/flavor_base/build_deps/packages/apt_get_packages:5]: 22.04?

File: flavors/template-Exasol-all-python-3.10/flavor_base/derived_from [hunk 2]
```diff
@@ -0,0 +1 @@
+flavors/python-3.8-minimal-EXASOL-6.2.0
```
Comment (Nicoretti) [inline flavors/template-Exasol-all-python-3.10/flavor_base/derived_from:1]: Do we need to bump the Exasol version at some point?

**Review by Nicoretti — APPROVED** [review]

File: flavors/template-Exasol-all-python-3.10/flavor_base/derived_from (hunk 2 above)
Comment (tkilias) [reply flavors/template-Exasol-all-python-3.10/flavor_base/derived_from:1]: No this file is for indicating from which flavor we derived this one. We need this for the package diffs.

File: flavors/template-Exasol-all-python-3.10/flavor_base/build_deps/packages/apt_get_packages (hunk 1 above)
Comment (tkilias) [reply flavors/template-Exasol-all-python-3.10/flavor_base/build_deps/packages/apt_get_packages:5]: Ah, I know what happens, the versions are actually ignored in the Dockerfile, this is a bug, but I will fix this separately.

**Pull request merged**